Formula/sha1dc.rb

```diff
@@ -0,0 +1,27 @@
+class Sha1dc < Formula
+  desc "Tool to detect SHA-1 collisions in files, including SHAttered"
+  homepage "https://github.com/cr-marcstevens/sha1collisiondetection"
+  url "https://github.com/cr-marcstevens/sha1collisiondetection.git",
+      :revision => "cb10f50d20a1c6c9efbf526c5da510c492d595b9"
```

Comment:
There isn't one — and of course we're in a world of relatively bleeding-edge software, what with SHAttered only being released less than a week ago.